test/unittests/state_transition_create_test.cpp

```diff
@@ -49,3 +49,74 @@ TEST_F(state_transition, create2_max_nonce)
 
     expect.post[*tx.to].nonce = pre.get(*tx.to).nonce;  // Nonce is unchanged.
 }
+
+TEST_F(state_transition, create_tx_deploying_eof)
+{
+    static constexpr auto create_address = 0x3442a1dec1e72f337007125aa67221498cdd759d_address;
```

Comment:
You can use `compute_create_address()`. But also you can omit `expect.post[create_address].exists = false` because it is the default.

Reply:
I'll leave expectation for explicitness.